design-tokens/Brand/Altinn.json

```diff
@@ -1 +1,130 @@
-{}
\ No newline at end of file
+{
+  "brand": {
+    "primary": {
+      "100": {
+        "value": "#feefef",
+        "type": "color",
+        "description": "AAA 12.8 on grey 800\nAA 6.4 as bkdg for interaction blue"
```

Comment:
Hva betyr "bkdg"?

Reply:
According to Google, "bkdg" is short for "BIOS and Kernel Developers Guide" 😂